tests/test_generator.py

```diff
@@ -213,19 +234,85 @@ def test_generate_configure(fileutils, wlmutils, generator_instance):
         "1200": "120",
         "VALID": "valid",
     }
-    app = Application("name_1", "python", file_parameters=param_dict)
-    app.attach_generator_files(to_configure=tagged_files)
-    job = Job(app, launch_settings)
-
-    # Call Generator.generate_job
-    path, _, _ = generator_instance.generate_job(job, 0)
-    # Retrieve the list of configured files in the test directory
-    configured_files = sorted(glob(str(path) + "/*"))
-    # Use filecmp.cmp to check that the corresponding files are equal
+    generator_instance._write_tagged_files(
+        files=files, params=param_set, dest=generator_instance.root
+    )
+    configured_files = sorted(glob(os.path.join(generator_instance.root) + "/*"))
```

Comment:
Nit: Did you mean
```suggestion
    configured_files = sorted(glob(os.path.join(generator_instance.root, "*")))
```
?